cmd/rest/main.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/go-devs-ua/octagon/migration"
 )
 
+const (
+	lvlDebug = "DEBUG"
```

Comment:
These constants should be in the same package where our logger is.

Reply:
@vpakhuchyi   it is already fixed